src/simulation/dynamics/Thrusters/thrusterStateEffector/thrusterStateEffector.rst

```diff
@@ -54,14 +54,23 @@ While the value of :math:`\kappa_i` for each thruster is computed numerically by
 
 where :math:`\kappa_{0,i}` corresponds to the initial conditions of the ``thrusterFactor`` variable for the i-th particular thruster.
 
-If the thruster is added to an auxiliary body :math:`F` connect to the main body :math:`B`, then this is accomplished
-using::
+If the thruster is added to an auxiliary body :math:`F`, of type :ref:`prescribedMotionStateEffector`,
+which is connected to the main spacecraft body :math:`B`, then this is accomplished using::
 
     thrusterSet.addThruster(thruster1, bodyStatesMsg)
 
-where ``thruster1`` is the thruster being added, and ``bodyStatesMsg`` is the state output message of
+Here ``thruster1`` is the thruster being added, and ``bodyStatesMsg`` is the state output message of
 the platform :math:`F`.  In the ``Update()`` routine the position of the platform :math:`F` relative
-to math:`B` is recomputed each time step.  If the thruster is added
+to :math:`B` is recomputed each time step.
+
+If the thruster is attached to the hub, then all output message variables that
+contain ``_B`` are written in the spacecraft hub :math:`B` frame as usual.
+However, if the thruster is attached to a :ref:`prescribedMotionStateEffector`,
+then some output message variables are written in the :math:`F` frame described above.
+In this case the prescribed motion body frame is still labeled generically with ``_B`` suffix
+as they are in the body frame, just the body frame of the prescribed motion platform, not
+the spacecraft body frame. These message variables include ``thrDir_B``, ``thrLoc_B``
+and ``thrusterDirection``, which are written in the prescribed motion state effector :math:`F` frame.
```

Comment:
What I'm seeing is that when the thruster is attached to a moving body, ``thrDir_B``, ``thrLoc_B`` are indeed expressed in the ``B`` frame, whereas ``thrusterDirection`` is implicitly defined in the local frame (aka platform frame ``F``, in this case). So I think what you just wrote is incorrect.

Reply:
Ok, thanks for the clarification.  I'll try to rewrite and push draft language.